Mathlib/CategoryTheory/Abelian/GrothendieckCategory/Colim.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Abelian.Basic
+import Mathlib.CategoryTheory.Abelian.GrothendieckAxioms.Basic
+
+/-!
+# Exactness of colimits in abelian categories
+
+Assuming `HasExactColimitsOfShape J C`, we rephrase the exactness of
+`colim : (J ⥤ C) ⥤ C` by saying that if `S : ShortComplex (J ⥤ C)`
+is exact, and we have colimit cocones `c₁`, `c₂`, `c₃`
+for `S.X₁`, `S.X₂` and `S.X₃`, then the induced short complex
+`c₁.pt ⟶ c₂.pt ⟶ c₃.pt` is exact.
+
+-/
+
+universe v v' u u'
+
+namespace CategoryTheory.Limits
+
+variable {C : Type u} [Category.{v} C] [Abelian C]
+  {J : Type u'} [Category.{v'} J]
+  [HasColimitsOfShape J C] [HasExactColimitsOfShape J C]
+  (S : ShortComplex (J ⥤ C)) (hS : S.Exact)
+  (c₁ : Cocone S.X₁) (hc₁ : IsColimit c₁) (c₂ : Cocone S.X₂) (hc₂ : IsColimit c₂)
+  (c₃ : Cocone S.X₃) (hc₃ : IsColimit c₃)
```

Comment:
We usually make the cocones implicit if there is an `IsColimit` assumption, is that inconvenient in your applications?